<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayScene</class>
 <widget class="QMainWindow" name="PlayScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="displayWindow" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="mainWindow" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="operateWindow" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="talkArea" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="diceArea" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionreturn"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsave">
   <property name="text">
    <string>保存游戏</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionreturn">
   <property name="text">
    <string>返回主界面</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
